### ISSUE TITLE
Issues with "Help file language"

### ISSUE THREAD

**Issue by NicolasSandCastle** [EWSoftware/SHFB#294]
Hello,

I use SandcastleBuilderGUI 2015.10.10.0.

I tried to change the "Help file language" and build it.

I first tried in French and German and it failed to build the help file. See the enclosed logs files.

[LastBuild - French.log.txt](https://github.com/EWSoftware/SHFB/files/247703/LastBuild.-.French.log.txt)
[LastBuild - German.log.txt](https://github.com/EWSoftware/SHFB/files/247702/LastBuild.-.German.log.txt)

I then tried in Italian. I tought that the labels in the outputed CHM would have been displayed in Italian, but it was in English:

------------------
![constructors](https://cloud.githubusercontent.com/assets/16655729/14997311/25629b9c-114c-11e6-93dd-a296f455a521.jpg)
![missing summary documentation](https://cloud.githubusercontent.com/assets/16655729/14997314/25651bf6-114c-11e6-9ea3-49f0ea20ab54.jpg)
![properties](https://cloud.githubusercontent.com/assets/16655729/14997312/2563c166-114c-11e6-9aff-bb7c6487dd56.jpg)
![syntax](https://cloud.githubusercontent.com/assets/16655729/14997313/2563ae1a-114c-11e6-8571-efad22456b7c.jpg)

------------------

However it seems to be OK in Spanish:

------------------
![constructores](https://cloud.githubusercontent.com/assets/16655729/14997402/9509c74a-114c-11e6-9066-4e7b87fe8ead.jpg)
![falta documentacion summary para](https://cloud.githubusercontent.com/assets/16655729/14997401/95073de0-114c-11e6-9f7c-477b16ccbe4e.jpg)
![metodos](https://cloud.githubusercontent.com/assets/16655729/14997398/94ffd758-114c-11e6-8de1-8eebc196471c.jpg)
![propiedades](https://cloud.githubusercontent.com/assets/16655729/14997400/950438de-114c-11e6-89cf-df46fe538dda.jpg)
![sintaxis](https://cloud.githubusercontent.com/assets/16655729/14997399/9503fefa-114c-11e6-9257-7de5f9f52d66.jpg)

------------------

What is wrong?

Best regards,

Nicolas

**Comment by EWSoftware** [comment]
Languages for the resource elements are only provided for some languages, not all.  See the [Localizing Language Resource Files](http://ewsoftware.github.io/SHFB/html/e2ed6222-c3ee-4c6e-b0f9-fa3c86bc3e2c.htm) help topic for details.

**Comment by NicolasSandCastle** [comment]
Hello Eric,

Thank you for your answer.

I saw this page about "Localizing Languages Ressources Files".

As I stated, I tried to build the help file in French and German. I saw that there is in the following folders the ressource files:

C:\Program Files (x86)\EWSoftware\Sandcastle Help File Builder\PresentationStyles\VS2013\Content\fr-FR
C:\Program Files (x86)\EWSoftware\Sandcastle Help File Builder\PresentationStyles\VS2013\Content\de-DE

However an error occured during the build, as you can see in the logs of my previous comment. Have you any idea what is wrong?

It should work since it is OK in Spanish. There is a "es-ES" folder with the ressource files. 

I tought there was an Italian folder, but there isn't. So it's OK if nothing translates. It's my mistake. Sorry.

Thanks again for helping.

Best regards,

Nicolas

**Comment by EWSoftware** [comment]
The French build is using the 2010 style which doesn't have French translations.  However, the error there is related to an invalid XML character in the .NET Framework XML comments which do have French versions available.  The file that's causing the issue is *C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\fr\System.Data.Services.Client.xml*.  Open that file, and fix the invalid character.  If the error message is accurate, it's on line 9, position 17.  This is a known issue but it's up to Microsoft to fix it.  Until that happens, you need to manually fix any such errors in the framework XML comments files.

For the German build, the encoding on the SHFB files is incorrect in the version you are using. Upgrade to the latest release to fix that problem.

**Comment by NicolasSandCastle** [comment]
Hello Eric,

Thank you very much.

I was able to build with "French (France)" as language and "VS2013" as the presentation style.

There was many hexadecimal characters in the System.Data.Services.Client.xml file. I opened it with ANSI encoding, than I saved with UTF-8 encoding (as the other files in the directory). It fixes my issue.

I will upgrade to the latest release if I want the German build.

Thank you for helping me.

Have a nice day!
